src/Bridge/Doctrine/Common/Filter/OrderFilterTrait.php

```diff
@@ -52,6 +52,10 @@ public function getDescription(string $resourceClass): array
                 'property' => $propertyName,
                 'type' => 'string',
                 'required' => false,
+                'enum' => [
```

Comment:
Not sure about this... :x

Reply:
If we do this, I don't think the UI would understand it.

Reply:
No, this has nothing to do with the UI. I'm talking about what we return from `getDescription`. It's the job of the `DocumentationNormalizer` to convert that into Swagger / OpenAPI docs, for example.

Reply:
And in fact, in Swagger / OpenAPI, we're indeed doing:

```yaml
schema:
    enum:
```

Reply:
In the swagger v3 we do :
```
schema:
   type: string
   enum: ...
```
so i felt like getDescription should not return nested arrays as it would mean that type is as well misplaced. Or should `type` be part of nested array indexed `schema` returned by getDescription ?

Reply:
@fredericbarthelet I think the confusion comes from treating `getDescription` as if it's meant to map to Swagger / OpenAPI directly. It's not.

We are returning some metadata about a filter, and it's the job of the respective `DocumentationNormalizer` to convert that into what's appropriate for their format.

Reply:
What we can do in the Swagger / OpenAPI `DocumentationNormalizer` though, is to take the `schema` value from the filter's `getDescription`, if it exists. Otherwise, we generate the `schema` value like the existing code does.

It means, from `OrderFilter`, we have to return:

```php
[
    'property' => $propertyName,
    'type' => 'string',
    'required' => false,
    'schema' => [
        'type' => 'string',
        'enum' => [
            strtolower(OrderFilterInterface::DIRECTION_ASC),
            strtolower(OrderFilterInterface::DIRECTION_DESC),
        ],
    ],
]
```

WDYT?

Reply:
you don't mind the double `'type' => 'string'` declaration ? Even if `schema` declaration would then take over `type` declaration, you still have to declare both as `type` is required for filter documentation computation.

Or should I modify `DocumentationNormalizer` so that `type` is no more required and have this output from `getDescription` : 

```
    'property' => $propertyName,
    'required' => false,
    'schema' => [
        'type' => 'string',
        'enum' => [
            strtolower(OrderFilterInterface::DIRECTION_ASC),
            strtolower(OrderFilterInterface::DIRECTION_DESC),
        ],
    ],
]
```

Reply:
I don't mind the duplication. You should be able to provide the `type` without providing `schema`. Also, we cannot change this because of BC.

Reply:
OK, i'll update my PR accordingly. Will let you know when it's done :)

Reply:
@teohhanhui changes made. Let me know if I got it right this time ;)